torchtext/transforms.py

```diff
@@ -294,6 +294,16 @@ class GPT2BPETokenizer(Module):
     def __init__(self, encoder_json_path: str, vocab_bpe_path: str, return_tokens: bool = False) -> None:
         super().__init__()
         self._seperator = "\u0001"
+        self.SPECIAL_TOKENS_ATTRIBUTES = [
```

Comment:
These special tokens are fairly common across all tokenizers. Would it make sense to pull these out into `commons` or `utils` so that we can reuse them across multiple transforms? cc @Nayef211

Reply:
I moved `SPECIAL_TOKENS_ATTRIBUTES` to `utils` in latest commit.

Reply:
@mthrok taught me about the [YAGNI principle](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) which I believe is quite applicable here. Let's not move these out into `commons` or `utils` until we see a concrete case where they can be reused. I also think it would make more sense for `SPECIAL_TOKENS_ATTRIBUTES` to be a class variable of `GPT2BPETokenizer` rather than an instance variable as these attributes would not be changing across instances. Lmk if this makes sense.